services/web/scripts/icomoon-import

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+ZIP_FILE=$1
```

Comment:
can this be document how icons create be added, like in /services/web readme

Reply:
Ah yes I still need to update the README ... will do

Reply:
Ok abandoning this script for now but added a note to document this... would like to bring it back but later.